recipes/libraw/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "libraw" %}
+{% set version = "0.19.5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LibRaw/LibRaw/archive/{{ version }}.tar.gz
+  sha256: 9a2a40418e4fb0ab908f6d384ff6f9075f4431f8e3d79a0e44e5a6ea9e75abdc
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - automake
+    - libtool
+    - pkg-config
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+    - lcms2
+    - jpeg
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  run:
+    - {{ pin_compatible('lcms2', max_pin='x.x') }}
+
+about:
+  home: http://www.libraw.org/
+  license: LGPL3
+  license_file: LICENSE.LGPL
```

Comment:
You should have `license_family` as an entry here too

Reply:
:+1: